build.gradle

```diff
@@ -672,6 +672,12 @@ project(':streams') {
     testCompile libs.junit
   }
 
+  configurations {
+    // manually exclude unnecessary zkclient transitive dependencies, remove this after KIP-4
+    compile.exclude module: 'jline'
+    compile.exclude module: 'netty'
+  }
+
```

Comment:
We could, but I wasn't sure if there was a reason why we used global exclusions in `core`, so I followed the same approach. In a sense global exclusions are less brittle as the dependencies can sometimes come from different places (eg zookeeper versus zkclient).